en/08_Changelogs/6.0.0.md

```diff
@@ -939,20 +958,31 @@ The related `onAfterRenewToken` extension hook has been renamed to `onAfterRenew
+SS_DATABASE_SERVER_REPLICA_01="read-only-server1"
+SS_DATABASE_SERVER_REPLICA_02="read-only-server2"
+#...
+SS_DATABASE_SERVER_REPLICA_12="read-only-server12"
+```
 
-See [read-only database replicas](/developer_guides/performance/read_only_database_replicas/) for more details.
+Replicas cannot define different configuration values for `SS_DATABASE_CLASS`, `SS_DATABASE_NAME`, or `SS_DATABASE_CHOOSE_NAME`. Those are restricted to prevent strange issues that could arise from having inconsistent database configurations across replicas.
+
+The `SS_DATABASE_SERVER`, `SS_DATABASE_USERNAME`, and `SS_DATABASE_PASSWORD` must be defined for each replica. If a port other than the default should be used, `SS_DATABASE_PORT` must also be defined for each replica.
+
+If one or more read-only replicas have been configured, then for each request one of the read-only replicas will be randomly selected from the pool of available replicas to handle queries for the rest of the request cycle, unless criteria has been met to use the primary database instead, for example a write operation.
 
 When replicas are configured, calling the method [`DB::get_conn()`](api:SilverStripe\ORM\DB::get_conn()) will now give a replica by default if one is able to be used. To get the primary database connection, call `DB::get_conn(DB::CONN_PRIMARY)` instead.
 
-Note that the `DB::CONN_PRIMARY` constant, which has a value of "primary", is used to specify the primary database used. Prior to CMS 6 when there was no DB replica support, the primary database was referred to as "default". If you have code that uses the string "default" to refer to the primary database, you should update it to use the `DB::CONN_PRIMARY` constant instead.
+Note that the `DB::CONN_PRIMARY` constant is used to specify the primary database used, which is the database that will be used for write operations. Prior to CMS 6 when there was no DB replica support, the primary database was referred to as "default". If you have code that uses the string "default" to refer to the primary database, you should update it to use the `DB::CONN_PRIMARY` constant instead.
 
 Note that some [`DataQuery`](api:SilverStripe\ORM\DataQuery) methods such as [`DataQuery::execute()`](api:SilverStripe\ORM\DataQuery::execute()) now work slightly differently as they will use the replica database if the queried `DataObject` has the [`DataObject.must_use_primary_db`](api:SilverStripe\ORM\DataObject->must_use_primary_db) configuration set to `true`. However calling the equivalent [`SQLSelect`](api:SilverStripe\ORM\Queries\SQLSelect) method via a `DataQuery` e.g. `$dataQuery->query()->execute()` will not respect the `DataObject.must_use_primary_db` configuration.
 
+See [read-only database replicas](/developer_guides/performance/read_only_database_replicas/) for more details.
```

Comment:
Moved because where it was felt like it was in the wrong place - it felt like this was two sections rolled into one since these are _usually_ at the end of a section.